third_party/openthread/CMakeLists.txt

```
@@ -76,6 +76,12 @@ if (OTBR_SRP_ADVERTISING_PROXY)
     set(OT_EXTERNAL_HEAP ON CACHE STRING "enable external heap" FORCE)
 endif()
 
+if (OTBR_ENABLE_EPSKC)
```

```suggestion
if (OTBR_EPSKC)
```

On the other hand, what about making a CMake option like `OT_EPSKC` in OT? In this way that you can use a `set` call to implement the logic.

---

Thanks! Accept.

---

> On the other hand, what about making a CMake option like OT_EPSKC in OT? In this way that you can use a set call to implement the logic.

Seems this is not done yet. Could you take a look?

---

Took care of it in https://github.com/openthread/openthread/pull/10034.

---

Thanks for adding the above CL